### PR TITLE
Ft cleanup deploy previews & and namespace manifests

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Refactors the remove deploy previews workflow
- Removes namespace resource limits

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

**Comment by NoahNsimbe** [comment]
@123MwanjeMike this PR has merge conflicts

**Comment by 123MwanjeMike** [comment]
> @123MwanjeMike this PR has merge conflicts

Thanks @NoahNsimbe 
These have been resolved

**Review by Baalmart — APPROVED** [review]
LGTM to me....

File: .github/workflows/remove-deploy-previews.yml [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+      remove_preview__events_check: ${{ steps.check_files.outputs.remove_preview__events_check }} # event checks
+      remove_preview__notifications: ${{ steps.check_files.outputs.remove_preview__notifications }} # notification
+      remove_preview__kafka_cluster_operator: ${{ steps.check_files.outputs.remove_preview__kafka_cluster_operator }} # kafka cluster operator
+      remove_preview__kafka_cluster: ${{ steps.check_files.outputs.remove_preview__kafka_cluster }} # kafka cluster
+      remove_preview__kafka_topics: ${{ steps.check_files.outputs.remove_preview__kafka_topics }} # kafka topics
+      remove_preview__airflow: ${{ steps.check_files.outputs.remove_preview__airflow }} # airflow
+      remove_preview__network_uptime: ${{ steps.check_files.outputs.remove_preview__network_uptime }} # network_uptime
+      remove_preview__channel_data: ${{ steps.check_files.outputs.remove_preview__channel_data }} # channel data job
+      remove_preview__message_broker_topics: ${{ steps.check_files.outputs.remove_preview__message_broker_topics }} # message broker topics
```
Comment (NoahNsimbe) [inline .github/workflows/remove-deploy-previews.yml:50]: Hi @123MwanjeMike. `remove_preview__message_broker_topics` might not be useful

File: .github/workflows/remove-deploy-previews.yml [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+      remove_preview__device_uptime: ${{ steps.check_files.outputs.remove_preview__device_uptime }} # device uptime
+      remove_preview__device_status: ${{ steps.check_files.outputs.remove_preview__device_status }} # device status
+      remove_preview__locate: ${{ steps.check_files.outputs.remove_preview__locate }} # locate
+      remove_preview__predict: ${{ steps.check_files.outputs.remove_preview__predict }} # predict
+      remove_preview__gp_model: ${{ steps.check_files.outputs.remove_preview__gp_model }} # gp_model
+      remove_preview__exceedances: ${{ steps.check_files.outputs.remove_preview__exceedances }} # exceedances
+      remove_preview__datawarehouse: ${{ steps.check_files.outputs.remove_preview__datawarehouse }} # datawarehouse
+      remove_preview__view: ${{ steps.check_files.outputs.remove_preview__view }} # view
+      remove_preview__calibrate: ${{ steps.check_files.outputs.remove_preview__calibrate }} # calibrate
+      remove_preview__average_jobs: ${{ steps.check_files.outputs.remove_preview__average_jobs }} # average and calibration jobs
```
Comment (NoahNsimbe) [inline .github/workflows/remove-deploy-previews.yml:41]: Hi @123MwanjeMike. `remove_preview__average_jobs` no longer exists

**Review by NoahNsimbe — CHANGES_REQUESTED** [review]

File: .github/workflows/remove-deploy-previews.yml (hunk 1 above)
Comment (123MwanjeMike) [reply .github/workflows/remove-deploy-previews.yml:50]: Thanks @NoahNsimbe 
Let me address these right away

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**